model-optimizer/mo/front/extractor.py

```diff
@@ -934,8 +945,9 @@ def input_user_data_repack(graph: Graph, input_user_shapes: [None, list, dict, n
         new_phs = _freeze_new_placeholder[node_id]
         if node_id not in _input_shapes:
             raise Error(
-                'Shape is not specified for the placeholder with name {} through --input_shape option.'.format(new_phs[0]['name']))
-        _ins = _input_shapes[node_id] # list
+                'Shape is not specified for the placeholder with name {} through --input_shape option.'.format(
```

Comment:
nit: can we split the text line into two lines, so the `format` and all its parameters will be in a one line?